cfgov/unprocessed/js/modules/util/dom-traverse.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+require( '../polyfill/class-list' );
+
+/**
+ * Get the sibling nodes of a dom node.
+ *
+ * @param {HTMLNode} elem - DOM element to get siblings of.
+ * @param {string} clazz - CSS classname.
+ * @returns {Array} List of sibling nodes.
+ */
+function getSiblings( elem, clazz ) {
```

Comment:
Yes! Fixed in https://github.com/cfpb/cfgov-refresh/commit/882b8e61c495b945b8b4c68301fb3f45d3f03248